db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_09_12_144508) do
+ActiveRecord::Schema[7.0].define(version: 2023_09_21_082209) do
```

Comment:
```suggestion
ActiveRecord::Schema[7.0].define(version: 2023_09_21_081147) do
```

This is currently referring to the other migration that got deleted.

Reply:
switched to 2023_09_12_081147

Reply:
Ah, I think I got this wrong actually, I mistook the date there for the other migration but actually it's one that's ready in main, sorry! You can revert this change back to how it was.

Reply:
I think it should be `2023_09_12_144508` actually as that's the latest migration in `main` (the migration in this file comes earlier than that one).